OW-744: Update RTC_OBJC_TYPE(RTCVideoDepthDataDelegate) protocol

joliveraortega opened the pull request.

gmininberg commented on Unity/ios/VonageUnityIntegration/ViewController.mm:
```diff
@@ -34,7 +34,13 @@ bool compress(const std::unique_ptr<uint8_t[]>& inputArray,
+        return Holographic::Compression::compress(depthDataMap, outputArray, outputSize);
+    }
+
     bool decompress(const uint8_t* inputArray,
```
with the patch i sent you for webrtc this can be removed since we dont need it.

gmininberg commented on Unity/ios/VonageUnityIntegration/ViewController.mm:
```diff
@@ -34,7 +34,13 @@ bool compress(const std::unique_ptr<uint8_t[]>& inputArray,
     }
-    
+
+    bool compress(CVPixelBufferRef depthDataMap,
```
with the patch i sent you for webrtc we can remove the other function.

joliveraortega replied:
Removed it.

gmininberg reviewed (commented):
@joliveraortega when i was trying to test this branch the app was crashing inside the compress function.

joliveraortega commented:
@gmininberg I addressed your review comments. Please, let know if we can take this.

> @joliveraortega when i was trying to test this branch the app was crashing inside the compress function.

Interesting, not sure what's different in my case. Let's see it together. Thanks.
